### ISSUE TITLE
Turkish and Arabic Translations for Miele-LXIV

### ISSUE THREAD

**Issue by zdavatz** [bettar/miele-lxiv#55]
These are the translation files exported from XCode at commit cebe7b7dd49bfa6a7d2a0ec4c1e2b9af2ad98938

[ar.xliff.zip](https://github.com/bettar/miele-lxiv/files/4280087/ar.xliff.zip)
[tr.xliff.zip](https://github.com/bettar/miele-lxiv/files/4280088/tr.xliff.zip)

**Comment by zdavatz** [comment]
Here are some considerations why Miele-LXIV should be translated into Arabic and Turkish 

Arabic language
Arabic 23 countries use Arabic as the official language as a "standard" which called "Fusha", All of these countries don't use delicate in official media, press, and education. Here is a list of Arabic speaking countries.

 Arabic in medical and healthcare
As I have graduated from university, I didn't study a single word of medicine in Arabic, as our primary medical language is English. 

This is going also for most of the Arabic countries except "Syria" where they study in Arabic (2 Universities). Eastern Arabic countries study Medicine in English as Egypt, Libya, Palestine, Iraq, Yemen, Saudi Arabia, UAE, Qatar, and several others, and Western Arabic countries study it in French like Tunisia, Algeria, and Morocco. Therefore we have no problem dealing with medical terms in English and French, However, the medical management in several countries like Gulf area (Saudi Arabia, Qatar, UAE, and Bahrain) require Arabic ready reports and translated solutions, because most of the managers and officials are not doctors and they use the Arabic language.

 I have many friends who work as doctors in 9 Arabic countries, most of them are my classmates, They did confirm authorities and management consider the Arabic language even though if the doctors are not using it. (Saudi Arabia, Qatar, and Bahrain) 

Turkish language

 In Turkey, doctors study medicine in Turkish, In all of the universities here. Turkish is the official language in healthcare here, especially for reporting. Most of the doctors have weak foreign language skills. Also, Turkish people do trust the applications, solutions, and services that speak their language. I have many doctor friends who work in the private sector as well as for the official healthcare services which are managed by gov. 

On a side note, I have been working in medical tourism for the private sector for some years now in Turkey managing communication and integration for many oncology cases and rare surgical cases. I have witnessed an increase in macOS usage among doctors especially the new generation and doctors who are educated aboard (Germany, USA, UK, and Canada), However, their numbers are limited. 


The macOS case

 Apple products are not popular among doctors in both Arabic countries and Turkey. However, It's gaining ground every day. Some private hospitals are using it, and many doctors like their stability. However, macOS users, in general, are lazy and they require everything to be ready for them like a program that speaks and supports their language (Turkey). Some hospitals in private sectors are using macOS and Linux as some of my friends reported and I have seen in (Turkey), but I have no idea about indications, usage, and numbers. If you have any question don't hesitate to ask. 

For the Arabic language, I believe decision-makers in certain countries will consider using the application if it supports Arabic because it'll be more management-friendly as well as system-friendly. Especially as there are some companies in the Gulf area (where the rich Arabic countries are) supports macOS and many companies provide macOS/ Apple support as well.
